### PR TITLE
Implement TaskTemplateWardPreview

### LINKED ISSUE #319: Ward Task Template Preview
Tasks - overview of wards in an organization

### PR DESCRIPTION
**Pull request by DasProffi**
closes #319

### CODE REVIEW

File: tasks/components/TaskTemplateWardView.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+const defaultComponentTranslation: Record<Languages, ComponentTranslation> = {
+  en: {
+    showAllTaskTemplates: 'Show all Task Templates',
+    taskTemplates: (numberOfTemplates) => `Task Templates (${numberOfTemplates})`
+  },
+  de: {
+    showAllTaskTemplates: 'Alle Task Vorlagen anzeigen',
+    taskTemplates: (numberOfTemplates) => `Task Vorlagen (${numberOfTemplates})`
```
Comment (use-to) [inline tasks/components/TaskTemplateWardView.tsx:23]: Do we stick with `Task Templates` as german translation? @MaxSchaefer

**Review by use-to — CHANGES_REQUESTED** [review]

File: tasks/components/TaskTemplateWardView.tsx
Comment (use-to) [inline tasks/components/TaskTemplateWardView.tsx:1]: Lets call it `TaskTemplateWardPreview`

File: tasks/components/TaskTemplateWardView.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <Button
+          className={tw('w-auto')}
+          onClick={() => router.push('/ward/templates')}
+        >
+          {translation.showAllTaskTemplates}
+        </Button>
+      </div>
+      <div className={tw('grid grid-cols-3 gap-4')}>
+        {taskTemplates.map((taskTemplate, index) => (
+          <TaskTemplateCard
```
Comment (use-to) [inline tasks/components/TaskTemplateWardView.tsx:64]: Make it not editable

File: tasks/components/TaskTemplateWardView.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            key={index}
+            name={taskTemplate.name}
+            subtaskCount={taskTemplate.subtasks.length}
+            onTileClick={() => { /* TODO do something onClick */ }}
```
Comment (use-to) [inline tasks/components/TaskTemplateWardView.tsx:68]: Navigate to task templates and automatically select the given card template.

**Review by use-to — CHANGES_REQUESTED** [review]

File: tasks/components/TaskTemplateWardView.tsx (hunk 3 above)
Comment (DasProffi) [reply tasks/components/TaskTemplateWardView.tsx:68]: this issue depends on #339

**Review by use-to — APPROVED** [review]

**Review by Paul-B98 — APPROVED** [review]

**Pull request merged**